stories/Organism/Footer/footer.scss

```diff
@@ -0,0 +1,128 @@
+@import "../../assets/scss/variables";
+@import "../../assets/scss/mixins";
+@import "../../assets/scss/base";
+
+.footer {
+  background-color: $color-gray-100;
+  padding: $spacing-48 $spacing-12 $spacing-40;
+  @include devicebreak(medium) {
+    padding: 52px 0 $spacing-48;  
+  }
+  &__wrapper {
+    margin: 0 auto;
+  }
+  p {
+    margin: 0;
+    font-size: $font-size-14;
+    line-height: 1.4;
+    @include devicebreak(medium) {
+      font-size: $font-size-16;
+    }
+  }
+
+  &__newsletter {
+    margin-top: $spacing-24;
+    @include devicebreak(medium) {
+    margin-top: 0;
+    }
+  }
+  &__head {
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: $spacing-48;
+    margin-bottom: $spacing-32;
+    border-bottom: 1px solid $color-gray-700;
+    @include devicebreak(medium) {
+      margin-bottom: $spacing-48;
+    }
+    &--two {
+      padding-bottom: $spacing-32;
+      @include devicebreak(medium) {
+        padding-bottom: $spacing-48;
+      }
+    }
+  }
+  &__bottom {
+    &--two {
+      flex-direction: column-reverse;
+      @include devicebreak(medium) {
+        flex-direction: unset;
+      }
```

Comment:
@Shubham-Sharma-77 If you want to set the flex direction till small then you can use that media query